src/js/components/Layout/v2/Subsection.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+import { RiArrowDownSLine, RiArrowUpSLine } from 'react-icons/ri';
+
+import Translate from 'utils/Translate';
+
+const Subsection = ({
+  title,
+  collapsable,
+  children,
+}) => {
+  // If a subsection is not collapsable, it is always expanded
+  // (collapsable: false --> expanded: true)
+  // If a subsection is collapsable, it is not expanded by default
+  // (collapsable: true --> expanded: false)
+  const [expanded, setExpanded] = useState(!collapsable);
+
+  const triggerCollapse = () => {
+    setExpanded(!expanded);
+  };
```

Comment:
instead of a single `collapsable` prop I would also add `isExpanded: boolean` as an initial value, that way we can control if the section is collapsed or not for both states when it is **collapsable** or **not collapsable**, so something like:

```jsx
const Subsection = ({
  title,
  isExpanded,
  collapsable,
  children,
}) => ...

...
Subsection.defaultProps = {
  isExpanded: true,
  collapsable: true,
};
```

```jsx
const [expanded, setExpanded] = useState(isExpanded);
```

prevent any toggle action if section is not collapsible, maybe name `disabled` fro this prop sound better but collapsable should also be good.
```jsx
const triggerCollapse = () => {
  if (collapsable) {
   setExpanded(!expanded);
  }
};
```

Reply:
Currently we are fine, because the `onClick` method would be `null` if a subsection is not collapseable, but I will add it just in case we would refactor something in the future.

Reply:
I've added additional prop `expandedByDefault` as you suggested, to make it more reusable in different scenarios. I didn't decide to go for `isExpanded`, not to mix `isExpanded` with `expanded`, as it might cause confusions.